nativelink-util/src/fs.rs

```diff
@@ -273,14 +273,8 @@ pub async fn create_dir_all(path: impl AsRef<Path>) -> Result<(), Error> {
 
 #[cfg(target_family = "unix")]
 pub async fn symlink(src: impl AsRef<Path>, dst: impl AsRef<Path>) -> Result<(), Error> {
-    let src = src.as_ref().to_owned();
-    let dst = dst.as_ref().to_owned();
-    call_with_permit(move |_| {
-        tokio::runtime::Handle::current()
-            .block_on(tokio::fs::symlink(src, dst))
-            .map_err(Into::<Error>::into)
-    })
-    .await
+    let _permit = get_permit().await?;
+    tokio::fs::symlink(src, dst).await.map_err(Into::into)
```

Comment:
wow. so simple. you are very much appreciated.

Reply:
> if many symlinks fire concurrently, you might still hit permit waits.
> 
> It would be good to know what scale of project you ran into this bug (I'm guessing Chromium), but I assume sufficiently large.

yes. I mentioned in the bug report that I was cross building Chromium with siso, when it will upload and reconstruct the whole macOS SDK and standard libc++.